### PR TITLE
Import script for Rochdale (closes #2199)

### LINKED ISSUE #2199: Import E08000005-Rochdale Metropolitan Borough Council
Xpress DC

### PR DESCRIPTION
**Pull request by GeoWill**

### CODE REVIEW

File: polling_stations/apps/data_collection/management/commands/import_rochdale.py [hunk 1]
```diff
@@ -4,21 +4,15 @@
...
-        # Mobile Unit at Black Dog Pub
-        if record.polling_place_id == "2829":
-            record = record._replace(polling_place_postcode="OL12 7JG")
-            rec = super().station_record_to_dict(record)
-            rec["location"] = Point(-2.186986, 53.631252, srid=4326)
-            return rec
-
-        # Bowls Pavilion
+        # Bowls Pavilion - Postcode gives wrong geocode
         if record.polling_place_id == "2577":
```
Comment (chris48s) [inline polling_stations/apps/data_collection/management/commands/import_rochdale.py]: Is this definitely ID 2577 again?
Usually the IDs change between elections with Xpress

File: polling_stations/apps/data_collection/management/commands/import_rochdale.py [hunk 2]
```diff
@@ -49,22 +43,28 @@ def address_record_to_dict(self, record):
...
         if uprn in ["23095362", "23095226"]:
             rec["postcode"] = "OL12 9NE"
 
+        # Not quite right, safer to throw away
+        if record.addressline6.strip() in [
+            "OL16 3BX",
+            "OL16 5SJ",
+            "M24 2YS",
+            "M245TG",
+            "M245TN",
```
Comment (chris48s) [inline polling_stations/apps/data_collection/management/commands/import_rochdale.py]: I don't think these last 2 are actually going to match anything. Usually the postcodes in `record.addressline6` (the input file) will have a space in them.

File: polling_stations/apps/data_collection/management/commands/import_rochdale.py (hunk 1 above)
Comment (GeoWill) [reply polling_stations/apps/data_collection/management/commands/import_rochdale.py]: Good catch. Thanks.

File: polling_stations/apps/data_collection/management/commands/import_rochdale.py (hunk 2 above)
Comment (GeoWill) [reply polling_stations/apps/data_collection/management/commands/import_rochdale.py]: ...and again!

**Comment by GeoWill** [comment]
Should be better now.

**Pull request merged**